packages/web/static/locales/en/dev.json

```diff
@@ -68,7 +68,7 @@
     "text":
       "Send or Request Celo Dollar easily with anyone in your address book, powered by decentralized phone number verification",
     "nonCustodial": "Non-custodial wallet",
-    "mobileUltra": "Mobile ultralight client",
+    "mobileUltra": "Mobile lightest client",
```

Comment:
hope we're certain we won't go lighter in the future, not leaving a lot of room with this new name haha ;)

Reply:
superduperlight?